src/frontend/screens/ProjectSettings/index.tsx

```diff
@@ -163,7 +159,7 @@ export const ProjectSettings = () => {
           }
           buttonText={formatMessage(m.viewTeam)}
           onPress={() =>
-            navigate('YourTeam', {projectName: projectInfo.projectName})
+            navigate('YourTeam', {projectName: projectInfo.projectName!})
```

Comment:
projectName is actually not used in the YourTeam Page, so we can take out that nav param completely